### ISSUE TITLE
Avoid double comment

### ISSUE THREAD

**Issue by DavidGOrtega** [iterative/cml#254]
After #236 if with workflows acts in on ```push``` and on ```pull_request``` the same comment appears one nested in the other in GH. Also its not nice to have the same comment twice.
Ideally we should check if the comment exists (via hash comparison) and avoid the comment.
We could setup a ```--force``` just in case we still wanted to do it.

**Comment by DavidGOrtega** [comment]
Not relevant
